lib/components/gene/artworks.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+'use strict'
+
+import Relay from 'react-relay'
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+
+import Separator from '../separator'
+import SerifText from '../text/serif'
+import ArtworksGrid from '../artwork_grids/infinite_scroll_grid'
+
+
+class Artworks extends React.Component {
+  static propTypes: Object = {
+    gene: React.PropTypes.shape({
+      id: React.PropTypes.string,
+    }),
+    medium: React.PropTypes.string,
+  }
+
+  render() {
+   return (
+      <ArtworksGrid artworks={[]} queryForState={this.stateQuery}  queryForPage={this.resolveQuery} queryArtworksKeypath="filter_artworks.hits" />
+      )
+  }
+
+  stateQuery = () => { {} }
+
+  resolveQuery = (component: ArtworksGrid, page: number, state: any) : string =>
+    Artworks.artworksQuery(this.props.gene.id, null, page)
+
+  static artworksQuery = (geneID: string, medium: ?string, page: number) => {
+    const mediumParam = medium ? ', medium: ' + medium : ''
```

Comment:
Might look a bit nicer to build up the params for the query as an array and then join them with `,`, just a stylistic thing.